README.md

```diff
@@ -9,16 +9,25 @@
 
 # The Open Source Observability Distribution
 
-Features that are not guarded behind a paywall.
-Opstrace installs secure, horizontally-scalable, open source observability in your cloud account combining open APIs with reduced toil.
-Opstrace is:
+Opstrace is a secure, horizontally-scalable, open source observability platform installed in your cloud account.
```

Comment:
I found "Opstrace installs .. observability" to be semantically weird for a long time: one cannot install an abstract concept like observability, just like one cannot install happiness :-).

What do you think?

Reply:
👍